testsuite/MDAnalysisTests/core/test_universe.py

```diff
@@ -498,6 +498,15 @@ def test_slicing_step_with_start_stop():
         assert_equal(universe.trajectory.timeseries(universe.atoms).shape,
                      (3341, 10, 3),
                      err_msg="Unexpected shape of trajectory timeseries")
+    @staticmethod
+    def test_slicing_step_dt():
+        universe = MDAnalysis.Universe(PDB_small, DCD)
+        times=[ts.time for ts in universe.trajectory]
+        universe.transfer_to_memory(step=2)
+        times2=[ts.time for ts in universe.trajectory]
+        assert_almost_equal(times[::2],times2,
```

Comment:
Thanks, I will go through everything and clean up. Guess I still have some fortran in me. :)